query-graphs/src/tree-rendering.ts

```diff
@@ -594,12 +636,86 @@ export function drawQueryTree(target: HTMLElement, treeData: TreeDescription) {
     if (crosslinks !== undefined && crosslinks.length) {
         treeText += buildPropertyList({crosslinks: crosslinks.length});
     }
-    d3selection
-        .select(target)
+    infoCard
         .append("div")
         .classed("qg-tree-label", true)
         .html(treeText);
 
+    // Add toolbar
+    function buildToolbarButton(id: string, description: string) {
+        return `<div id='${id}-button' class='qg-toolbar-button'>
```

Comment:
same issue with IDs. This won't work for the JupyterLab plugin.

I would recommend to instead use something like 
```
const toolbar = infoCard.append("div").classed("qp-toolbar");
function addToolbarButton(symbol: string, description: string, action: ()=>void) {
    toolbar
       .append("div")
       .classed("qg-toolbar-button")
       .on("click", action)
       .html(`<span class="qg-toolbar-icon">
    <svg viewbox='-8 -8 16 16' width='20px' height='20px'>
      <use href='#${id}-symbol' class="qg-collapsed" />
    </svg>
  </span>
<span class="qg-toolbar-tooltip">${description}</span>`);
}
addToolbarButton("zoom-in", "Zoom In", () =>  zoomBehavior.scaleBy(baseSvg.transition().duration(200), zoomFactor));
```

Reply:
I like the idea of passing the action function to avoid the ids